keyboards/acheron/shark/alpha/readme.md

```diff
@@ -0,0 +1,17 @@
+# Acheron Aχξρων 40-SM-O-MX-TH-WI (Codename "SharkPCB") QMK firmware
+
+<p align="center">
+  <img align="middle" src="https://raw.githubusercontent.com/Gondolindrim/acheronLibrary/master/graphics/acheronLong.png"  width="400"> 
+</p>
```

Comment:
Please remove the HTML tagts, and use markdown, instead. 

Also, please rehost on Imgur or the like.

Reply:
This shouldn't be here — this README was superseded by the one on the `shark` folder. This has been deleted on 9d939d252c